src/commands/env.rs

```diff
@@ -62,13 +75,15 @@ impl Command for Env {
             .shell
             .or_else(&infer_shell)
             .ok_or(Error::CantInferShell)?;
+
         let multishell_path = make_symlink(config);
         let binary_path = if cfg!(windows) {
             multishell_path.clone()
         } else {
             multishell_path.join("bin")
         };
-        println!("{}", shell.path(&binary_path));
+        let merged_path = merge_path(&binary_path)?;
+        println!("{}", shell.set_env_var("PATH", merged_path.as_str()));
```

Comment:
I think that not all shells are using the same env var expression. But I'm not sure. I don't really know why we had `shell.path(..)` instead of `set_env_var` 😁

Reply:
i'm not sure either 😁

Reply:
Seems that tests fail. Also, seems like Fish has a different way of adding a PATH entry

Reply:
@Schniz Fixed